project-koku/koku
# [COST-2505] track ocp on cloud update time

Fixes [COST-2505](https://issues.redhat.com/browse/COST-2505)

Requires https://gitlab.cee.redhat.com/insights-qe/iqe-cost-management-plugin/-/merge_requests/1267 as well

Changes proposed in this PR:
* Add timestamp for OCP on Cloud summary to the sources stats endpoint
*

Testing Instructions:
1. Check out cost-2505-track-ocp-on-cloud-update-time for iqe-cost-management plugin in conjunction with this PR and run some OCP on cloud tests to make sure they complete and wait for ocp on cloud to finish calculating before running



---
Additional Context:

This does change the format of the stats endpoint, but the MR for the plugin modifies helpers to account for the new format


```
GET /api/cost-management/v1/sources/eb234bb8-4ebb-42e2-9241-a1355be44504/stats/
HTTP 200 OK
Allow: GET, HEAD
Cache-Control: max-age=0, no-cache, no-store, must-revalidate, private
Content-Type: application/json
Expires: Thu, 21 Apr 2022 20:44:24 GMT
Vary: Accept

{
    "provider_linked": true,
    "data_updated_date": "2022-04-21 20:30:53",
    "ocp_on_cloud_data_updated_date": "2022-04-21 20:31:39",
    "2022-04-01": {
        "manifests": [
            {
                "assembly_id": "6388c6fb-e02e-4070-887e-955d575b141d",
                "billing_period_start": "2022-04-01",
                "files_processed": "1/1",
                "last_process_start_date": "2022-04-21 20:29:46",
                "last_process_complete_date": "2022-04-21 20:29:55",
                "last_manifest_complete_date": "2022-04-21 20:30:40",
                "manifest_modified_datetime": "2022-04-21 20:26:46"
            }
        ],
        "ocp_on_cloud": [
            {
                "ocp_source_uuid": "c202e995-0920-43be-8306-33c521b35227",
                "ocp_on_cloud_updated_datetime": "2022-04-21 20:31:07"
            }
        ]
    },
    "2022-03-01": {
        "manifests": [
            {
                "assembly_id": "adb458af-adac-4dea-b8ae-f3177a610723",
                "billing_period_start": "2022-03-01",
                "files_processed": "1/1",
                "last_process_start_date": "2022-04-21 20:29:57",
                "last_process_complete_date": "2022-04-21 20:30:00",
                "last_manifest_complete_date": "2022-04-21 20:30:53",
                "manifest_modified_datetime": "2022-04-21 20:26:44"
            }
        ],
        "ocp_on_cloud": [
            {
                "ocp_source_uuid": "c202e995-0920-43be-8306-33c521b35227",
                "ocp_on_cloud_updated_datetime": "2022-04-21 20:31:39"
            }
        ]
    }
}
```
